k2/csrc/array_ops_inl.h

```diff
@@ -104,6 +105,29 @@ void ExclusiveSumPerRow(const Array2<T> &src, Array2<T> *dest) {
   }
 }
 
+// called in RandUniformArray1
+template <typename T, typename std::enable_if<
+                          std::is_floating_point<T>::value>::type * = nullptr>
```

Comment:
People will usually not use the default type `void`. It took me a while to figure out
that the default type is `void`.

Reply:
added `T` as the default type